references/classification/presets.py

```diff
@@ -17,7 +17,10 @@ def __init__(
         augmix_severity=3,
         random_erase_prob=0.0,
     ):
-        trans = [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
+        trans = [
+            transforms.PILToTensor(),
```

Comment:
I can't move the `ConvertImageDtype` transform here, because AA in v1 only supports uint8 inputs. Are we benchmarking v2 with float32 or uint8?

Reply:
We are going to test both.